### PR TITLE
🇷🇺 language

### PR DESCRIPTION
**Pull request by xllifi**

### CODE REVIEW

File: src/main/resources/assets/ok-boomer/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    { "translate": "text.ok-boomer.prefix" }
+  ],
+  "text.config.ok-boomer.option.useCinematicCamera": "Использовать кинематографическую камеру при бумах",
+  "text.config.ok-boomer.option.enableScreenBooming": "Разрешить бумать экраны",
+  "text.config.ok-boomer.option.boomScrollSensitivity": "Чувствительность бумов к прокрутке",
+  "text.config.ok-boomer.option.boomTransition": "Анимация бума",
+  "text.config.ok-boomer.option.defaultBoom": "Бум по умолчанию",
+  "text.config.ok-boomer.option.boomTransitionSpeed": "Скорость анимации бума",
+  "text.config.ok-boomer.category.boomLimits": "Ограничения бума",
+  "text.config.ok-boomer.option.boomLimits.enableLimits": "Включить лимиты",
```
Comment (BasiqueEvangelist) [inline src/main/resources/assets/ok-boomer/lang/ru_ru.json:32]: ограничения would probably be nice here

File: src/main/resources/assets/ok-boomer/lang/ru_ru.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "text.config.ok-boomer.option.boomLimits.allowBoomingOut": "Разрешить негативный бум",
```
Comment (BasiqueEvangelist) [inline src/main/resources/assets/ok-boomer/lang/ru_ru.json]: negative is used here in the sense of inverted, so `отрицательный` would probably be better here

File: src/main/resources/assets/ok-boomer/lang/ru_ru.json (hunk 1 above)
Comment (xllifi) [reply src/main/resources/assets/ok-boomer/lang/ru_ru.json:32]: I prefer sticking to source length though
limits - 6 letters / 20 pixels ("i"=1, "t"=3)
лимиты - 6 letters / 31 pixels ("ы"=6)
ограничения - 11 letters / 65 pixels (!!)

File: src/main/resources/assets/ok-boomer/lang/ru_ru.json (hunk 2 above)
Comment (xllifi) [reply src/main/resources/assets/ok-boomer/lang/ru_ru.json]: yes👍

**Review by BasiqueEvangelist — APPROVED** [review]
Should be fine™️

**Pull request merged**